? changes
? foo.c
? hal-0.4.0-clean-on-startup.patch
? hal-0.4.0-fix-fstab-sync-logging.patch
? hal-0.4.0-fix-nonblock.patch
? hal-0.4.0-storage-policy-never-use-uuid.path
? hal-0.4.0.tar.gz
? hal-storage.pc
? intltool-extract
? intltool-extract.in
? intltool-merge
? intltool-merge.in
? intltool-update
? intltool-update.in
? mkinstalldirs
? doc/api/Doxyfile
? doc/api/Makefile
? doc/api/Makefile.in
? doc/api/html
? doc/conf/Makefile
? doc/conf/Makefile.in
? doc/spec/Makefile
? doc/spec/Makefile.in
? doc/spec/hal-spec.xml
? doc/spec/spec.tar
? fdi/30osvendor/Makefile
? fdi/30osvendor/Makefile.in
? fdi/40oem/Makefile
? fdi/40oem/Makefile.in
? fdi/50user/Makefile
? fdi/50user/Makefile.in
? fdi/90defaultpolicy/Makefile
? fdi/90defaultpolicy/Makefile.in
? fdi/95userpolicy/Makefile
? fdi/95userpolicy/Makefile.in
? hald/foo
? hald/hout
? hald/sout
? libhal-storage/.deps
? libhal-storage/.libs
? libhal-storage/Makefile
? libhal-storage/Makefile.in
? libhal-storage/libhal-storage.la
? libhal-storage/libhal-storage.lo
? po/Makefile
? po/Makefile.in
? po/Makefile.in.in
? po/POTFILES
? po/da.gmo
? po/de.gmo
? po/fr.gmo
? po/hal.pot
? po/nl.gmo
? tools/foo
? tools/fstab-sync
? tools/fstab-sync.8
? tools/linux/hal-hotplug-map
Index: tools/fstab-sync.c
===================================================================
RCS file: /cvs/hal/hal/tools/fstab-sync.c,v
retrieving revision 1.24
diff -u -p -r1.24 fstab-sync.c
--- tools/fstab-sync.c	15 Oct 2004 00:45:52 -0000	1.24
+++ tools/fstab-sync.c	15 Oct 2004 20:52:35 -0000
@@ -1322,6 +1322,7 @@ add_udi (const char *udi)
 	char *dir = NULL;
 	char *last_slash;
 	char *mount_point;
+	char *device_file;
 	
 	dir = strdup (_PATH_FSTAB); 	 
 	last_slash = strrchr (dir, '/'); 	 
@@ -1368,11 +1369,15 @@ add_udi (const char *udi)
 	
 	if (!create_mount_point_for_volume (mount_point))
 		goto error;
+
+	device_file = hal_device_get_property_string (hal_context, udi, "block.device");
 	
 	fstab_update_debug (_("%d: added mount point '%s' for device '%s'\n"),
-			    pid, mount_point, "foo");
+			    pid, mount_point, device_file);
 	syslog (LOG_INFO, _("added mount point %s for %s"), 
-		mount_point, "foo");
+		mount_point, device_file);
+	
+	hal_free_string (device_file);
 	
 	close (fd);
 	
